src/main/java/crazypants/enderio/machine/obelisk/attractor/TileAttractor.java

```diff
@@ -143,8 +147,8 @@ public SpawnObeliskAction getSpawnObeliskAction() {
 
   private class Target extends FakePlayerEIO {
 
-    public Target() {
-      super(getWorld(), getLocation(), new GameProfile(null, MachineObject.blockAttractor.getUnlocalisedName() + ":" + getLocation()));
+    public Target(World world) {
```

Comment:
What was the reason for this change?

Reply:
IDE was complaning that all the entities needed a World parameter

Reply:
This is true only for entities which are loaded from the world, this never happens for this class. But it's a harmless change so whatever.